src/os/bluestore/BlueStore.cc

```diff
@@ -5380,7 +5400,8 @@ int BlueStore::_balance_bluefs_freespace(PExtentVector *extents)
   // reclaim from bluefs?
   if (reclaim) {
     // round up to alloc size
-    reclaim = P2ROUNDUP(reclaim, cct->_conf->bluefs_alloc_size);
+    uint64_t alloc_size = bluefs->get_alloc_size(bluefs_shared_bdev);
+    reclaim = P2ROUNDUP(reclaim, alloc_size);
```

Comment:
If we move P2ROUNDUP inside BlueFS::reclaim_blocks then we could get rid of BlueFS::get_alloc_size.